modules/framework/test.py

```diff
@@ -0,0 +1,65 @@
+from modules.db.db import Db
+from modules.framework.timer import Timer
+from modules.framework.parser import DataProvider
+from modules.db.model.car import Car
+from pathlib import Path
+import json
+
+
+class Test:
+    def __init__(self, db: Db, data_file: str, results_filename: str):
+        self.parser = DataProvider(data_file)
+        self.results_filename = results_filename
+        self.db = db
+        
+        self.testing_points = [1e3, 1e4, 1e5, 1e6]
+        self.num_of_elements = 0
+        self.num_of_rows_in_iteration = 1000
```

Comment:
if variable is constant it should define as classwide variable:
class Test:
    NUM_OF_ROWS_IN_ITERATION = 1000
    itd